src/maps/southern_us/move.ts

```diff
@@ -0,0 +1,103 @@
+import { MoveValidator } from "../../engine/move/validator";
+import { MoveHelper } from "../../engine/move/helper";
+import { MoveData } from "../../engine/move/move";
+import { MoveAction } from "../../engine/move/move";
+import { Good, goodToString } from "../../engine/state/good";
+import { City } from "../../engine/map/city";
+import { inject } from "../../engine/framework/execution_context";
+import { InvalidInputError } from "../../utils/error";
+import { peek } from "../../utils/functions";
+import { PlayerData } from "../../engine/state/player";
+import { injectGrid } from "../../engine/game/state";
+import { Log } from "../../engine/game/log";
+import { PlayerHelper } from "../../engine/game/player";
+
+export class SouthernUSMoveHelper extends MoveHelper {
+  readonly COASTAL_CITIES = new Set([
+    "New Orleans",
+    "Mobile",
+    "Savannah",
+    "Charleston",
+  ]);
+
+  isWhiteInCostal(city: City, good: Good): boolean {
+    return good === Good.WHITE && this.COASTAL_CITIES.has(city.name());
+  }
+
+  canDeliverTo(city: City, good: Good): boolean {
+    if (this.isWhiteInCostal(city, good)) {
+      return true;
+    }
+
+    return super.canDeliverTo(city, good);
+  }
+}
+
+export class SouthernUSMoveValidator extends MoveValidator {
+  protected readonly grid = injectGrid();
+  protected readonly moveHelper = inject(SouthernUSMoveHelper);
+
+  validateEnd(action: MoveData): void {
```

Comment:
Rather than overriding validateEnd, I think it's less code to override MoveHelper.canDeliverTo